src/main/java/com/coveros/selenified/application/Check.java

```diff
@@ -468,20 +460,20 @@ default String checkConfirmationEquals(String expectedConfirmationText, double w
      * @param timeTook                    - the amount of time it took for wait for something (assuming we had to wait)
      * @return String: if the confirmation is present, the text of the confirmation
      */
-    default String checkConfirmationMatches(String expectedConfirmationPattern, double waitFor, double timeTook) {
+    String checkConfirmationMatches(String expectedConfirmationPattern, double waitFor, double timeTook) {
         // check for the object to be present
         String confirmation;
-        boolean isConfirmationPresent = getApp().is().confirmationPresent();
+        boolean isConfirmationPresent = this.app.is().confirmationPresent();
         if (isConfirmationPresent) {
-            confirmation = getApp().get().confirmation();
+            confirmation = this.app.get().confirmation();
         } else {
-            getReporter().fail("to find confirmation with the text matching pattern <b>" + expectedConfirmationPattern + ON_PAGE, waitFor, NO_CONFIRMATION, timeTook);
+            this.reporter.fail("to find confirmation with the text matching pattern <b>" + expectedConfirmationPattern + ON_PAGE, waitFor, NO_CONFIRMATION, timeTook);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "to find confirmation with the text matching pattern <b>" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)